### DISCUSSION TITLE
Package 'basic_topic_pkg' not found

### DISCUSSION THREAD

**Discussion by jjh946** [Road-Balance/gcamp_ros2_basic#6] [Q&A]
ros2 워크숍을 듣고 노션을 보고 기본내용들을 따라하고 있습니다.
NVIDIA Jetson Nano에서의 ROS 2 설치 & 리눅스 커멘드 기초 :  https://puzzling-cashew-c4c.notion.site/NVIDIA-Jetson-Nano-ROS-2-cdf045dc44e241ad93885d5e274cbcf4
ROS 2 기본 실습 : https://puzzling-cashew-c4c.notion.site/ROS-2-c0d986eb869441b2b1248f3a7a2936da
을 보면서 따라가고 있는데요, 
기본 실습에서 노드에 대한 예제를 하는 과정에서, 
cd ~/ros2_seminar_ws
cbp basic_topic_pkg
roseloq

ros2 run basic_topic_pkg  example_node_1

# 실행 결과
[INFO] [node_name]: 
==== Hello ROS 2 ====

을 실행하는데 에러가 납니다
![image](https://user-images.githubusercontent.com/85427212/137653385-a49bbb63-9508-46bd-8e6c-b427f3063db1.png)

사진과 같이 페키지가 없다고 하는데 제가 페키지를 잘못 건드린가 해서 모두 삭제하고  'NVIDIA Jetson Nano에서의 ROS 2 설치 & 리눅스 커멘드 기초'의 세팅을 다시 해봐도 같은 에러가 반복됩니다. 
![image](https://user-images.githubusercontent.com/85427212/137653509-43d6f856-86e2-4835-aade-9e003e0ff26c.png)
gui상으로 확인을 해봐도 basic_topic_pkg가 gcamp-ros2-semiar안에 하나,  basic_topic_pkg폴더 안에도 또 하나 있는데
자꾸 basic_topic_pkg페키지가 없다고 나옵니다

**Comment by kimsooyoung** [comment]
질문 주신 내용에 대한 더욱 정확한 답변을 위해 추가 정보를 요청드립니다.

`~/ros2_seminar_ws` 안의 폴더 구조를 위해 다음 명령어 입력 및 결과를 공유주시면 감사합니다.  

```
cd ~/ros2_seminar_ws
tree .
```

더불어, ROS 2 세미나에 대한 코드는 [다음](https://github.com/Road-Balance/gcamp-ros2-seminar) 링크에 위치하고 있어 추가 질문시에는 해당 Repo에서 open 해주시면 더욱 감사합니다.

**Reply by jjh946** [reply]
![image](https://user-images.githubusercontent.com/85427212/138199992-a95a62f8-6cb7-47e8-9e5d-952c07e9cf3c.png)

**Reply by jjh946** [reply]
일단 tree는 이와 같습니다

**Reply by kimsooyoung** [reply]
답변감사합니다. 강의 노트에 실수가 있었습니다.

ROS 2 패키지들은 `ros2_seminar_ws/src` 폴더에 위치해야 빌드 및 실행이 가능합니다. 
강의 내용 수정을 완료하셨으며 아래 커멘드 입력 후 다시 진행해보시기 바랍니다.

```bash
$ mv ~/Documents/gcamp-ros2-seminar ~/ros2_seminar_ws/src
```

**Reply by jjh946** [reply]
답변 감사합니다!
